File: Plugins/Aspose_Words_for_PHP/src/quickstart/findandreplace/php/FindAndReplace.php
<?php
/*
 * Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
 *
 * This file is part of Aspose.Words. The source code in this file
 * is only intended as a supplement to the documentation, and is provided
 * "as is", without warranty of any kind, either expressed or implied.
 */


class FindAndReplace {

    public static function replaceText() {

        // The path to the documents directory.
        $dataDir = "/usr/local/apache-tomcat-8.0.22/webapps/JavaBridge/Aspose_Words_Java_For_PHP/src/quickstart/findandreplace/data/";

        // Open the document.
        $doc = new Java("com.aspose.words.Document",$dataDir."ReplaceSimple.doc");
        // Check the text of the document
        echo "Original document text: " . $doc->getRange()->getText() . "<BR>";
        // Replace the text in the document.
        $doc->getRange()->replace("_CustomerName_", "James Bond", false, false);
        // Check the replacement was made.
        echo "Document text after replace: " . $doc->getRange()->getText();
        // Save the modified document.
        $doc->save($dataDir . "ReplaceSimple Out.doc");
    }

}

File: Plugins/Aspose_Words_Java_for_Ruby/lib/asposewordsjavaforruby/findandreplace.rb
module Asposewordsjavaforruby
  module FindAndReplace
    def initialize()
        # Find and replace text in document.
        replace_text()
    end
        
    def replace_text()
        # The path to the documents directory.
        data_dir = File.dirname(File.dirname(File.dirname(__FILE__))) + '/data/quickstart/'

        # Open the document.
        doc = Rjb::import('com.aspose.words.Document').new(data_dir + "ReplaceSimple.doc")

        # Check the text of the document.
        puts "Original document text: " + doc.getRange().getText()
        
        # Replace the text in the document.
        doc.getRange().replace("_CustomerName_", "James Bond", false, false)
        
        # Check the replacement was made.
        puts "Document text after replace: " + doc.getRange().getText()
        
        # Save the modified document.
        doc.save(data_dir + "ReplaceSimple Out.doc")
    end 

  end
end
